testing/xmrig/APKBUILD

```diff
@@ -0,0 +1,37 @@
+# Contributor: Oleg Titov <oleg.titov@gmail.com>
+# Mainteiner: Oleg Titov <oleg.titov@gmail.com>
+
+pkgname=xmrig
+pkgver=2.8.3
+pkgrel=0
+pkgdesc="XMRig is a high performance Monero (XMR) miner"
+url="https://github.com/xmrig/xmrig"
+arch="x86_64 x86 aarch64 armhf armv7"
+license="GPL-3.0"
+depends="libuv libmicrohttpd openssl"
+makedepends="cmake libmicrohttpd-dev libuv-dev openssl-dev"
+subpackages="$pkgname-doc"
+source="$pkgname-$pkgver.tar.gz::https://github.com/xmrig/xmrig/archive/v$pkgver.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
+
+build() {
+	cd "$builddir"
+	mkdir build
+	cd build
+	cmake ..
+	make
+}
+
+check() {
+	cd "$builddir"/build
+	./$pkgname --version || true
+}
+
+package() {
+	install -Dm 755 build/xmrig $pkgdir/usr/bin
```

Comment:
this makes broken package, should be `install -Dm 755 build/xmrig $pkgdir/usr/bin/xmrig`

Reply:
Fixed.